.github/workflows/coverage-reporter.yml

```diff
@@ -0,0 +1,31 @@
+---
+## Workflow to process coverage report
+## For now the coverage is not reported. This workflow is just here as preparation for
+## https://github.com/elastic/observability-robots/issues/1546
+name: coverage-reporter
+on:
+  workflow_run:
+    workflows:
+      - test
+    types:
+      - completed
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download artifacts
+        run: >
+          gh run download ${{ github.event.workflow_run.id }}
+          --name coverage
+          --repo "${GITHUB_REPOSITORY}" 
+          --dir coverage
+        env:
+          GH_TOKEN: ${{ github.token }}
+      - run: ls -l coverage/matrix_results
+#      - run: ./spec/scripts/coverage_converge.sh
```

Comment:
unrequired?

Reply:
I added this comment here: https://github.com/elastic/apm-agent-ruby/blob/359520fdce91f9c5065d7ca7fd674ab38fecad53/.github/workflows/coverage-reporter.yml#L2-L4